src/ontology/cl-edit.owl

```diff
@@ -30104,6 +30107,36 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4030011 "epithelial cell of
 AnnotationAssertion(rdfs:label obo:CL_4030011 "epithelial cell of proximal tubule segment 3")
 SubClassOf(obo:CL_4030011 obo:CL_0002306)
 
+# Class: obo:CL_4030012 (kidney loop of Henle short descending thin limb epithelial cell)
+
+AnnotationAssertion(Annotation(<http://purl.org/dc/elements/1.1/contributor> <http://orcid.org/0000-0003-2804-127X>) Annotation(oboInOwl:hasDbXref "PMID:33769951"^^xsd:string) obo:IAO_0000115 obo:CL_4030012 "Epithelial cell of the descending thin limb of the short loop (cortical) nephron limited to the outer medulla (mainly inner strip). It is known in some mammalian species that the short descending limb of the loop of Henle selectively expresses the serine protease Corin, the homeobox TF Uncx, and the urea channel Slc14a2."^^xsd:string)
```

Comment:
@bvarner-ebi looks like you'll need to run the script to remove xsd:strings, there are several in this PR. Thank you.

Reply:
I thought I did run it... I don't think it applies to the newly added terms but could be wrong. I will run it again though.